packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -2963,6 +2963,232 @@ void main() {
     expect(tester.getTopRight(find.text('hint')).dx, 760.0);
   });
 
+  testWidgets('FloatingLabelAlignment.toString()', (WidgetTester tester) async {
+    expect(FloatingLabelAlignment.start.toString(), 'FloatingLabelAlignment.start');
+    expect(FloatingLabelAlignment.center.toString(), 'FloatingLabelAlignment.center');
+  });
+
+  group('floatingLabelAlignment', () {
+    Widget buildID({required TextDirection textDirection,
```

Comment:
Nit: The naming is a bit confusing, with `buildID` being a special case of `buildInputDecorator`...  Would it be better as `buildInputDecoratorWithFloatingLabel` or something?